src/game/states/play.rs

```diff
@@ -531,7 +536,7 @@ impl GameState for Play {
                 }
             }
             Mouse(MouseButton::Right) => {
-                if let Some(gm) = self.world.player.wep.utilities.throw_grenade(ctx, &mut s.mplayer).unwrap() {
+                if let Some(gm) = self.world.player.wep.utilities.cock_grenade(ctx, &mut s.mplayer).unwrap() {
```

Comment:
cook, not cock.

Reply:
At least it's consequently called `cock` everywhere..

Reply:
Falch said that was what it was called...